### DISCUSSION TITLE
Glowroot 0.14 beta not capturing servlet 6.00 web transactions on tomcat 10.1.5

### DISCUSSION THREAD

**Discussion by gmtjlmartin** [glowroot/glowroot#976] [Q&A]
Hi folks,

I was wondering if anyone has gotten glowroot 0.14 beta to capture servlet requests using Spring 6.0 web mvc on tomcat 10.1.5?  I am catching the startup transactions; however, my web transactions are not being tracked at all. Anything special I can do to help figure out what is going on?  Can I set a higher level of logging in glowroot to capture more diagnostic information?  Any insight would be appreciated.

**Comment by robinroos** [comment]
Perhaps this is related to #972  #949  #950  #945 ?

I'd really like to see traction on these, but I also appreciate that noone is actually paid to move glowroot forward.

**Reply by gmtjlmartin** [reply]
I am starting to reconsider usage of glowroot moving forward. Probably time to seriously look at jumping to open telemetry. Pleasantly surprised on the amount of trace information obtained out of the box with the agent from:  https://github.com/open-telemetry/opentelemetry-java-instrumentation. I was able to pair the trace data emitted with Open Zipkin to view said traces and it gave me that warm and fuzzy feeling knowing I could once again have a tool that would help me find the slow requests and more importantly  a trace level breakdown for a tomcat 10.1.5 Spring Web MVC 6.x Jakarta EE web application. I think the pain point now is finding a good aggregator/collector and GUI that will provide the info glowroot's rich UI produces. My guess is unless others step in to move glowroot forward, its doomed to become "glowrott" as I have many times mistyped its name.  I must admit it pains me to think of this tool's demise as glowroot has been a godsend, but alas all good things must come to an end.

**Comment by nowheresly** [comment]
This PR https://github.com/glowroot/glowroot/pull/981 aims at providing support for apps using `jakarta.*` packages.

**Comment by nowheresly** [comment]
The last beta4 is able to capture servlet requests with spring 6 and the last tomcat. Feel free to test it and give your feedback.

**Reply by gmtjlmartin** [reply]
I have integrated the new release with my web app and glowroot works as advertised. 

Glowroot lives on! 

I truly appreciate the effort from the contributors to this project. 

I have been working with opentelemetry SDK for Java now for a few weeks and I am convinced that OTEL is the way of the future in this solution space. With that being said, the ease and completeness of the features in glowroot UI are not as easy to capture using OTEL. Traces and metrics are scattered and must be reassembled with various other elements of the JVM to provide a similar one stop UI experience. I totally get the need for OTEL especially with modern microservices architectures. Distributed tracing is simply a must have for monitoring in that solution space. 

The irony is I had just proclaimed glowroot dead and penned the ticket for my project to convert to OTEL yesterday.  Yet like magic a new release of glowroot has surfaced a year after its predecessor and it works for my project.
